### PR TITLE
Add a way for Lonestar to flex

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
Add 7 giant gems of different color, each gem is worth 1 million credit to buy and sell.
Crystals
[Sprites are a courtesy of ma9ici4n.](https://ma9ici4n.itch.io/gems)

### CODE REVIEW

**Comment by Kazkin** [comment]
Ok but why?

**Comment by r4d6** [comment]
> Ok but why?

I wanted to add a giant 64x64 diamond that would cost 1 million credits so that CEOs could flex the huge amount of credit they got.
Trilby told me she got it, but then made https://github.com/sojourn-13/sojourn-station/pull/1952, which is an anomalous crystal that's so valuable Lonestar can't find a buyer, which is the opposite of what I asked.
So I found those sprites on the internet and decided to do it myself.

TL;DR : It's a way for Lonestar to flex how much money they have by buying useless shit who's whole purpose is to allow flexing.
